packaging/Brocfile.js

```diff
@@ -42,22 +40,25 @@ var loader = pickFiles('bower_components', {srcDir: '/loader.js', destDir: '/'})
 var jsTree = mergeTrees([sourceTree, globalExports, loader]);
 
 // Transpile modules
-var compiled = compileES6(jsTree, {
-  wrapInEval: false,
-  loaderFile: 'loader.js',
-  inputFiles: ['ember-charts/**/*.js'],
-  ignoredModules: ['ember'],
-  outputFile: '/ember-charts.js',
-  legacyFilesToAppend: ['globals-output.js']
+var compiled = esTranspiler(jsTree, {
+  filterExtensions:['js', 'es6'],
+  stage: 0,
+  moduleIds: true,
+  modules: 'amd'
 });
 
 // Wrap in a function which is executed
-compiled = wrap(compiled);
+compiled = concat(compiled, {
+  inputFiles: [
+    '**/*.js'
+  ],
+  outputFile: '/ember-charts.js'
+});
+// compiled = wrap(compiled);
```

Comment:
Good catch, that wraps everything in a self executing function, we could probably live without it, but it doesn't hurt to wrap all the things in a closure.